lib/binding_web/test/language-test.js

```diff
@@ -36,7 +36,7 @@ describe('Language', () => {
     });
 
     it('handles invalid inputs', () => {
-      assert.equal(null, JavaScript.nodeTypeForId(-1));
+      assert.equal(null, JavaScript.nodeTypeForId(-12));
```

Comment:
shouldn't need this

Reply:
Currently, this test gives

```
AssertionError: expected null to equal 'ERROR'
```

this is because the `ts_builtin_sym_error` is equal to `-1`. Perhaps a separate PR should assert the node type id is nonnegative in the web bindings (all bindings?)

Reply:
Weird, I can't reproduce this at all, with your branch or master...

Reply:
Weird... I have been having some issues with my local test running so it is probably a quirk on my end. I have removed the change